rules/rules.libsonnet

```diff
@@ -101,6 +101,41 @@
           },
         ],
       },
+      {
+        name: 'noobaa-replication.rules',
+        rules: [
+          {
+            record: 'noobaa_replication_total_writes_size',
+            expr: |||
+              sum_over_time(sum by (replication_id) (NooBaa_replication_last_cycle_writes_size)[1y:30s])
```

Comment:
30 seconds is based on the scraping of Prometheus.
We need to make sure that this configuration is not changed.
Also, these sums need to calculate the whole history of the metric and not only a year back.
I would check if there is a possibility to get all history that exists for these metrics.
If not then I guess that we would need to take a large enough buffer and update it.